### PR TITLE
For #26844: Fix ktlint issues and remove them from baseline.

### LINKED ISSUE #26844: Update ktlint to 0.47.0
https://github.com/pinterest/ktlint/releases/tag/0.47.0

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21492)

### PR DESCRIPTION
**Pull request by mcarare**
### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [ ] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #26844

### CODE REVIEW

**Review by Amejia481 — APPROVED** [review]
🧹 ✨

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# .git-blame-ignore-revs
+# Fix issues from ktlint baseline
+4c6a1f5b46f84349e8405ef5b23dd1381106dc48
```
Comment (gabrielluong) [inline .git-blame-ignore-revs:3]: This is great!

Comment (gabrielluong) [inline .git-blame-ignore-revs:3]: I was double checking if the blame is ignored locally, but I was still seeing this in Android Studio. 

Screen Shot 2022-09-09 at 11 12 27 AM

Comment (mcarare) [reply .git-blame-ignore-revs:3]: This is for ignoring a commit when assigning blame, not for changing the git log.

Comment (gabrielluong) [reply .git-blame-ignore-revs:3]: Ignoring the log panel, you would see the blame in the sidebar for that line of code.

Comment (gabrielluong) [reply .git-blame-ignore-revs:3]: Sorry, I figured it out. You have to configure locally to specify the ignore file via `git config blame.ignoreRevsFile .git-blame-ignore-revs`

File: .git-blame-ignore-revs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# .git-blame-ignore-revs
```
Comment (gabrielluong) [inline .git-blame-ignore-revs:1]: This line seems unnecessary.

File: .git-blame-ignore-revs [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+# Fix issues from ktlint baseline
```
Comment (gabrielluong) [inline .git-blame-ignore-revs:2]: Perhaps we should also add the issue number here.

**Review by gabrielluong — APPROVED** [review]

**Comment by gabrielluong** [comment]
@mcarare Also, can't help but see that this PR is closing the issue for the gradle plugin upgrade? I assume that isn't intended.

File: .git-blame-ignore-revs (hunk 1 above)
Comment (mcarare) [reply .git-blame-ignore-revs:3]: Oh, ok! For that local git must be configured to use the file.

`git config --local blame.ignoreRevsFile .git-blame-ignore-revs`

should do the trick.

**Comment by mcarare** [comment]
> @mcarare Also, can't help but see that this PR is closing the issue for the gradle plugin upgrade? I assume that isn't intended.

No, I meant to reference the original issue for ktlin update, 26844.

File: .git-blame-ignore-revs (hunk 2 above)
Comment (mcarare) [reply .git-blame-ignore-revs:1]: I kept the format recommended in the GitHub [documentation](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view).

Comment (gabrielluong) [reply .git-blame-ignore-revs:1]: 👍

**Pull request merged**